databricks/koalas/series.py

```diff
@@ -24,6 +24,7 @@
 
 import numpy as np
 import pandas as pd
+from pandas.core.accessor import CachedAccessor
```

Comment:
Maybe inline the CachedAccessor code? Is it similar to our lazy property? If yes, can we reconcile the two?

Reply:
Yea, the purpose is similar but one additional stuff it does is that it creates an instance of `CachedAccessor.accessor` (here `DatetimeMethods`) with the owner (`Series`).

There's another difference. When we access it via statically (class method), it will return the class itself whereas our lazy property isn't.

With out lazy property we can do it more freely but can't do above stuff. If you mean to deduplicate, I am not sure if we can. It's basically just get and set but one is class and is function.

Do you want to port this into our code base too? It will be only used for accessors.

Reply:
Let's port this into our codebase then. It's ok to have some duplication.